docs/Get started/FAQ.md

```diff
@@ -127,6 +127,10 @@ Each host’s OS version is available using the `api/v1/fleet/hosts` API endpoin
 
 It’s possible in Fleet to retrieve each host’s kernel version, using the Fleet API, through `additional_queries`. The Fleet configuration options YAML file includes an `additional_queries` property that allows you to append custom query results to the host details returned by the `api/v1/fleet/hosts` endpoint. [Check out an example configuration file with the additional_queries field](https://fleetdm.com/docs/using-fleet/fleetctl-cli#fleet-configuration-options).
 
+### How does Fleet set the `name` attribute for scheduled queries?
+
+Query automations send data to your log destination with a `name` in this format: `pack_global_<your-query-name>`.
```

Comment:
Is there any special formatting happening with `<your-query-name>` (lowercasing, kebab-casing, stripping special characters, that sort of thing)? From the example I would _guess_ that it's being lowercased and formatted as kebab-case, but as a user I'd feel more confident inferring the `name` if that was explicitly spelled out.

Reply:
No special formatting is happening. Example pulled from my osquery results logs:

`"pack/Global/Get USB device type - 107 - Jul 17 17:47:18.000"`